cypress/e2e/buy-shirt.cy.ts

```diff
@@ -1,26 +1,40 @@
+import {MenuContentPage} from "../page/index";
+import {LoginPage} from "../page/index";
+import {AddressStepPage} from "../page/index";
+import {PaymentStepPage} from "../page/index";
+import {ProductsListPage} from "../page/index";
+import {ShippingStepPage} from "../page/index";
+import {ShopingCartPage} from "../page/index";
```

Comment:
you can rewrite this as:
```suggestion
import {
  MenuContentPage,
  LoginPage,
  AddressStepPage,
  PaymentStepPage,
  ProductsListPage,
  ShippingStepPage,
  ShopingCartPage
} from "../page/index";
```